catalog/app/utils/gtm.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react'
+
+import cfg from 'constants/config'
+
+function loadScript(src: string) {
+  const s = window.document.createElement('script')
+  s.type = 'text/javascript'
+  s.async = true
```

Comment:
i'd merge these two functions into one accepting a dict of attributes, so that it can be called like `addScript({ async: true, src: '...' })` and `addScript({ innerText: '...' })`, but that's up to you, since the code is good enough as it is

Reply:
It's tricky to support adding **any** script attribute, so I've added only `async`, `innerText`, `src`